index.js

```diff
@@ -31,6 +31,10 @@ if (process.env.hasOwnProperty('GCLOUD_DEBUG_LOGLEVEL')) {
 if (process.env.hasOwnProperty('GCLOUD_DEBUG_DISABLE')) {
   config.enabled = false;
 }
+if (process.env.hasOwnProperty('GCLOUD_DEBUG_RELATIVE_REPOSITORY_PATH')) {
```

Comment:
I'm happy to get rid of the environment variable, the hope was to let people configure it with whatever mechanism is most convenient since it doesn't introduce undue complexity to support both. If you'd like it to be config only I can remove the env var.